frame/support/src/unsigned.rs

```diff
@@ -87,7 +91,7 @@ macro_rules! impl_outer_validate_unsigned {
 				#[allow(unreachable_patterns)]
 				match call {
 					$( Call::$module(inner_call) => $module::validate_unsigned(source, inner_call), )*
-					_ => $crate::unsigned::UnknownTransaction::NoUnsignedValidator.into(),
+					_ => $crate::unsigned::InvalidTransaction::NotFullyValidated.into(),
```

Comment:
This is a breaking change. Previously if you made an unsigned call to non-existent module it would merely return `UnknownTransaction`, which is interpreted by the txpool as "not valid now, might be valid at some point in the future". Which is sort of ok, but it has undersirable consequence of not banning the transaction - so basically you can spam the txpool with (one; the same) transaction to non-existent module to make it constantly re-verify it.
I think it's more wise to reject such transaction (pretty much the same would happen if you try to call a non-existent function (we return `InvalidTransaction::Call`)